compiler/luci/service/src/Nodes/CircleReshape.cpp

```diff
@@ -139,7 +149,7 @@ loco::TensorShape Algorithm::visit(const luci::CircleReshape *node)
   for (uint32_t dim_index = 0; dim_index < output_shape.rank(); ++dim_index)
   {
     const uint32_t dim_value = output_shape.dim(dim_index).value();
-    if (static_cast<int>(dim_value) == -1)
+    if (output_shape.dim(dim_index).known() == false)
```

Comment:
(Optional)
```suggestion
    if (not output_shape.dim(dim_index).known())
```

Reply:
> (Optional)

This is recommended from https://github.com/Samsung/ONE/pull/13935#discussion_r1753130003

Reply:
> This is recommended from [#13935 (comment)](https://github.com/Samsung/ONE/pull/13935#discussion_r1753130003)

I'm fine with this line, then. 😁